windows/sound.cpp

```diff
@@ -60,6 +60,13 @@ QString CSound::LoadAndInitializeDriver ( QString strDriverName,
         return tr ( "The current selected audio device is no longer present in the system." );
     }
 
+    // Save number of channels from last driver
+    // Need to save these (but not the driver name) as CheckDeviceCapabilities() overwrites them
+

```

Comment:
Sorry, tiny one: I'd drop the blank lines between the comment and what it's commenting (it's "nearer" the block before it as is).  Also, then, only the one line after the declares.

Reply:
Ok. Please re check